radarr/umbrel-app.yml

```diff
@@ -18,10 +18,14 @@ gallery:
   - 2.jpg
   - 3.jpg
 path: ""
+releaseNotes: >-
+  Not sure where to find the releaseNotes for the update,
+  releases page on Github seem to be including not much in it.
+
```

Comment:
Ya, it looks like our only option is to mash something together from the changes shown in the Github releases. I would suggest that we just say something very simple about this update taking Radarr from 4.1.0.6175 all the way to v5.0.3.8127 and then mention any major changes that change the functionality of the app. For instance it looks like as of v5, UI authentication is mandatory. We can then link to the releases page.

Reply:
Okays, I updated the release notes !